pkg/infoschema/builder.go

```diff
@@ -954,9 +954,9 @@ func (b *Builder) createSchemaTablesForDB(di *model.DBInfo, tableFromMeta tableF
 	if b.enableV2 {
 		for name, id := range di.TableName2ID {
```

Comment:
I checked the assignment of `di.TableName2ID`, the value comes from `meta.GetAllNameToIDAndTheMustLoadedTableInfo`. And in that function the table name is matched by `L:` regexp. Should we match for `O:` instead? Otherwise `tableName: model.NewCIStr(name),` is still always lowercased

Reply:
Good point. We should change to `O:` match